runtime/jcl/common/jclcinit.c

```diff
@@ -41,6 +41,9 @@
 #include "omrversionstrings.h"
 #include "j9modron.h"
 #include "omr.h"
+#if defined(OPENJ9_BUILD)
```

Comment:
The required change to set OPENJ9_BUILD isn't added to the module.xml's so I don't think it will ever be defined.

Reply:
I will freely admit that I'm not clear on how or why that variable is only set on some of our builds and not others. Any advice?

Reply:
To fix it, see the following example copied from runtime/jcl/cl_se7_basic/module.xml which allows use of `#if defined(OPENJ9_BUILD)`. Add another line for jclcinit. All of jcl_se7_base, cl_se9, cl_se10 need to be changed .

```
			<makefilestub data="ifeq ($(OPENJ9_BUILD),true)"/>
			<makefilestub data="vm_scar$(UMA_DOT_O) : CFLAGS += -DOPENJ9_BUILD"/>
			<makefilestub data="endif"/>
```

Reply:
You'll need to do the similar thing in runtime/rasdump/module.xml for javadump.cpp

Reply:
Ahh, I see. Took me a minute to figure out vm_scar was a file. Doh!

Reply:
Done. I've also added notes to the javadump issue.